tests/recon/torch/modules/test_latent.py

```diff
@@ -0,0 +1,60 @@
+import torch
+import unittest
+from abc import ABC, abstractmethod
+from typing import Iterator
+import torch.nn as nn
+from bdpy.recon.torch.modules import latent as latent_module
+
+
+class DummyLatent(latent_module.BaseLatent):
+    def __init__(self):
+        self.latent = torch.tensor([1.0])
+
+    def reset_states(self):
+        self.latent = torch.zeros_like(self.latent)
```

Comment:
パラメーターの初期化時にインスタンスを新たに作成せずにデータをin-placeで書き換えたいので以下でお願いします

```suggestion
        with torch.no_grad():
            self.latent.fill_(0.0)
```